submissions/currency_converter/apifetch.js

```diff
@@ -0,0 +1,21 @@
+// import  APIKEY from "./key";
+
+const convert = () => { 
+    const countrySymbol = document.getElementById('symbol').value; 
+    const amountInUSD = document.getElementById('amount').value;
+
+const options = { 
+    method: 'GET',  
+    headers: { 
+        accept: 'application-json',
+        // 'X-API-KEY': APIKEY
+        'X-API-KEY' : "f53b14db7658d93aeb2f8589838b87e01d730d82b9c73e9878065452902790b7"
+    }
+};
+    const apiUrl = 'https://api.chimoney.io/v0.2/info/usd-amount-in-local?destinationCurrency=USD&amountInUSD=1' 
```

Comment:
What does this do?

Reply:
this is the endpoint for currency converter it is then passed to the fetch method

Reply:
IS this not converting USD to USD?

Reply:
it converts local to usd